device-types/Ubiquiti/UCK-G2.yml

```diff
@@ -1,7 +1,7 @@
 ---
 manufacturer: Ubiquiti
-model: UCK-G2
-slug: uck-g2
+model: Cloud Key
+slug: cloud-key
 part_number: uck-g2
```

Comment:
Please add `u_height: 0`

Reply:
It actually has a height of 1 and you can order a [mount kit](https://store.ui.com/collections/unifi-accessories/products/cloud-key-g2-rack-mount-accessory) to fit it in a rack.

Reply:
Yup, but the device itself isnt rack mountable without that adapter.. I think we should try to find a better way to model this for the case where you have a UCK-G2 or UCK-G2+ but no rack mount adapter

Maybe others can chime in on this and we can figure out a better option (or just make it a 1U device). Would you like to create a poll in the discussion board or would you like me to?

Reply:
If you don't mind, please setup the poll 👍

Reply:
The more I thought it over in my head the more complicated it became. Im retracting my previous opinion
Lets keep it with u_height 1 for now, and if someone comes up with a better way in the future we can open another PR for that